test/test_sensors.js

```diff
@@ -1,249 +1,46 @@
-import app           from './server';
-import db            from '../src/models/db';
-import should        from 'should';
-import supertest     from 'supertest';
-
-import * as Const from './constants';
-import * as Err   from '../src/errors';
-
-const server = supertest.agent(app);
-
-describe('Sensors API', () => {
-  describe('Preconditions', () => {
-    beforeEach(done => {
-      db().then(models => {
-        return Promise.all([
-          models.Sensors.destroy({ where: {} }),
-          models.Datastreams.destroy({ where: {} })
-        ]).then(() => done());
-      });
-    });
-
-    it('/Sensors endpoint should exist and be empty', done => {
-      server.get('/Sensors')
-      .expect('Content-Type', /json/)
-      .expect(200)
-      .end((err, res) => {
-        should.not.exist(err);
-        res.status.should.be.equal(200);
-        res.body[Const.iotCount].should.be.equal(0);
-        res.body.value.should.be.instanceof(Array).and.have.lengthOf(0);
-        done();
-      });
-    });
+import commonTests  from './common';
+import db           from '../src/models/db';
+import * as CONST   from './constants';
+
+const mandatory     = ['name', 'description', 'encodingType'];
+const optional      = ['metadata'];
+const associations  = ['Datastreams'];
+
+const tester = commonTests('Sensors', mandatory, optional, associations);
+
+[CONST.encodingTypes.UNKNOWN,
```

Comment:
Same thing, insert these tests inside a `describe` block. Maybe naming the block as `Sensors API - specific`